account_fiscal_position_partner_type/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Valentin Vinagre <valentin.vinagre@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    fiscal_position_type = fields.Selection(
+        selection=[("b2c", "Customer"), ("b2b", "Company")],
+        string="Fiscal Position Type",
+        default=lambda self: self.env.company.default_fiscal_position_type,
```

Comment:
is it possible to change this to a method?
```

        default=lambda self: self._default_fiscal_position_type(),
    )

    @api.model
    def _default_fiscal_position_type(self):
        return self.env.company.default_fiscal_position_type
```

Reply:
Done